resources/js/components/Listing/Filters/CustomSelected.vue

```diff
@@ -0,0 +1,60 @@
+<script>
+import { swatches } from '../../../stores/useSwatches'
+
+export default {
+    props: {
+        selectedValues: {
+            type: Object,
+            default: () => {},
+        },
+    },
+
+    render() {
+        return this.$scopedSlots.default(this)
+    },
+
+    computed: {
+        activeFilters() {
+            return Object.keys(this.selectedValues)
+                .filter((filterKey) => {
+                    return (
+                        this.selectedValues[filterKey].showFilter &&
+                        (Array.isArray(this.selectedValues[filterKey].value)
+                            ? this.selectedValues[filterKey].value.length
+                            : !!this.selectedValues[filterKey].value)
+                    )
+                })
+                .flatMap((filterKey) => {
+                    let result
+
+                    if (
+                        Array.isArray(this.selectedValues[filterKey].value) &&
+                        !['category', 'price', ...Object.keys(swatches.value)].includes(filterKey)
+                    ) {
+                        result = this.selectedValues[filterKey].value.map((selected) => selected)
+                    }
+
+                    if (filterKey === 'price') {
+                        result = price(this.selectedValues[filterKey].value[0]) + ' - ' + price(this.selectedValues[filterKey].value[1])
+                    }
+
+                    
+                    if (Object.keys(swatches.value).includes(filterKey)) {
```

Comment:
I removed the Color config @royduin . Every swatch can be used as a "active" filter now. Even though color filters are circles with the color as background. I think the label will suffice as "active filter" ( E.G. "Blue") as most webshops do this.